examples/api/README.md

```diff
@@ -123,18 +130,20 @@ context that the analyzer uses.
 
 ## Writing Tests
 
-Examples are required to have tests. There is already a "smoke test" that runs
-all the API examples, just to make sure that they start up without crashing. In
-addition, we also require writing tests of functionality in the examples, and
-generally just do what we normally do for writing tests. The one thing that
-makes it more challenging for the examples is that they can't really be written
-for testability in any obvious way, since that would probably complicate the
-example and make it harder to explain.
+Examples are required to have tests. There is already a "smoke test" that simply
+builds and runs all the API examples, just to make sure that they start up
+without crashing. In addition, we also require writing tests of functionality in
+the examples, and generally just do what we normally do for writing tests. The
+one thing that makes it more challenging for the examples is that they can't
+really be written for testability in any obvious way, since that would probably
+complicate the example and make it harder to explain.
 
 As an example, in regular framework code, you might include a parameter for a
 `Platform` object that can be overridden by a test to supply a dummy platform,
 but in the example, this would be unnecessary complexity for the example. In all
```

Comment:
This would be unnecessarily complex for an example.

Reply:
Done.